docs/dev-guide/design/extension-sandboxing.md

```
@@ -0,0 +1,165 @@
+| `net.http` | `fetch` intents accepted | host executes; extension never opens a socket       |
+| `net.git`  | `clone` intents accepted | host executes via `go-git`                          |
+| `net.raw`  | direct socket access     | **tier 2 only**; cannot be granted to a WASM module |
+
+There is no filesystem capability. Filesystem access is always mediated by the
+plan.
+
+## The shim
+
+The host re-executes its own binary as a hidden subcommand:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)